fv3core/stencils/d_sw.py

```diff
@@ -546,30 +546,53 @@ def __init__(self, namelist, column_namelist):
         self.delnflux_nosg_w = DelnFluxNoSG(self._column_namelist["nord_w"])
         self.delnflux_nosg_v = DelnFluxNoSG(self._column_namelist["nord_v"])
         self.fvtp2d_dp = FiniteVolumeTransport(
-            namelist,
-            namelist.hord_dp,
-            self._column_namelist["nord_v"],
-            self._column_namelist["damp_vt"],
+            grid=GridIndexing.from_legacy_grid(self.grid),
```

Comment:
lol, the name 'legacy' grid is funny.

Reply:
I could just name it `from_grid` if you like, but I'm not sure whether that's clear.

Reply:
You can leave it, I just laughed when I read it :)